tests/testSpherePoint.py

```diff
@@ -123,16 +123,21 @@ def testInit1ArgErrors(self):
         SpherePoint(afwGeom.Point3D(0.0, 0.0, 1.0))
         SpherePoint(afwGeom.Point3D(42.78, -46.29, 38.27))
 
+    def testCopyConstructor(self):
+        sp = SpherePoint(-42.0*degrees, 45.0*degrees)
+        spcopy = SpherePoint(sp)
+        self.assertEqual(sp, spcopy)
+
     def testInitNArgFail(self):
         """Tests if only 1- or 2-argument initializers are allowed.
         """
-        with self.assertRaises(NotImplementedError):
+        with self.assertRaises(TypeError):
             SpherePoint()
-        with self.assertRaises(NotImplementedError):
+        with self.assertRaises(TypeError):
             SpherePoint("Rotund", "Bovine")
-        with self.assertRaises(NotImplementedError):
+        with self.assertRaises(TypeError):
             SpherePoint(42)
-        with self.assertRaises(NotImplementedError):
+        with self.assertRaises(TypeError):
             SpherePoint("ICRS", 34.0, -56.0)
```

Comment:
I'm also not very happy about this, but didn't see an easy way to restore the old behavior. Fortunately production code usually tests for `Exception`.